flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/slot/TaskSlotTableImpl.java

```diff
@@ -134,6 +143,12 @@ public TaskSlotTableImpl(
 		slotActions = null;
 		state = State.CREATED;
 		closingFuture = new CompletableFuture<>();
+
+		asyncExecutor = new ThreadPoolExecutor(
```

Comment:
What spoke against using the `ioExecutor` in `TaskManagerSharedServices` when creating the `TaskSlotTableImpl`?

Reply:
I think we can indeed use the `ioExecutor`